tests/Agent/IntegrationTests/Applications/BasicMvcApplication/BasicMvcApplication.csproj

```diff
@@ -30,6 +30,7 @@
     <DefineConstants>DEBUG;TRACE</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
+    <LangVersion>default</LangVersion>
```

Comment:
Is this required?  My reading of https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/configure-language-version is that it will default to this setting automatically if it is not present.

Reply:
It was a necessary change so that I could use the "modern" `using` declaration syntax over in https://github.com/newrelic/newrelic-dotnet-agent/pull/1616/files#diff-30592c3c36f251b029ff1e2ba1b7fe563b9225c6d41c106e55b32b6ec76ab3acR66 -- I have no idea why it doesn't default to `default` by default... ;-)